### ISSUE TITLE
build problem ? 

### ISSUE THREAD

**Issue by hughhugh** [kyamagu/mexopencv#283]
**Hi, I  met this problem when make . What  can I do to solve this issue? Thank you.**

**system ubuntu 12.04 64bit , matlab 2013a, opencv 2.4.13, g++ 4.4**
**Log as follows:**

```
name@ubuntu:~/mexopencv$ make MATLABDIR=/usr/local/MATLAB/R2013a
/usr/local/MATLAB/R2013a/bin/mex -c -largeArrayDims -cxx -Iinclude -I/usr/local/include/opencv -I/usr/local/include    -outdir lib/ src/mexopencv_features2d.cpp
In file included from include/mexopencv_features2d.hpp:17,
                 from src/mexopencv_features2d.cpp:9:
include/mexopencv.hpp:88: error: ‘DIST_USER’ is not a member of ‘cv’
include/mexopencv.hpp:89: error: ‘DIST_L1’ is not a member of ‘cv’
include/mexopencv.hpp:90: error: ‘DIST_L2’ is not a member of ‘cv’
include/mexopencv.hpp:91: error: ‘DIST_C’ is not a member of ‘cv’
include/mexopencv.hpp:92: error: ‘DIST_L12’ is not a member of ‘cv’
include/mexopencv.hpp:93: error: ‘DIST_FAIR’ is not a member of ‘cv’
include/mexopencv.hpp:94: error: ‘DIST_WELSCH’ is not a member of ‘cv’
include/mexopencv.hpp:95: error: ‘DIST_HUBER’ is not a member of ‘cv’
include/mexopencv.hpp:99: error: ‘DIST_USER’ is not a member of ‘cv’
include/mexopencv.hpp:100: error: ‘DIST_L1’ is not a member of ‘cv’
include/mexopencv.hpp:101: error: ‘DIST_L2’ is not a member of ‘cv’
include/mexopencv.hpp:102: error: ‘DIST_C’ is not a member of ‘cv’
include/mexopencv.hpp:103: error: ‘DIST_L12’ is not a member of ‘cv’
include/mexopencv.hpp:104: error: ‘DIST_FAIR’ is not a member of ‘cv’
include/mexopencv.hpp:105: error: ‘DIST_WELSCH’ is not a member of ‘cv’
include/mexopencv.hpp:106: error: ‘DIST_HUBER’ is not a member of ‘cv’
include/mexopencv.hpp:110: error: ‘LINE_4’ is not a member of ‘cv’
include/mexopencv.hpp:111: error: ‘LINE_8’ is not a member of ‘cv’
include/mexopencv.hpp:112: error: ‘LINE_AA’ is not a member of ‘cv’
include/mexopencv.hpp:116: error: ‘FILLED’ is not a member of ‘cv’
In file included from src/mexopencv_features2d.cpp:9:
include/mexopencv_features2d.hpp:51: error: ‘KAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:52: error: ‘cv::KAZE’ has not been declared
include/mexopencv_features2d.hpp:53: error: ‘cv::KAZE’ has not been declared
include/mexopencv_features2d.hpp:54: error: ‘cv::KAZE’ has not been declared
include/mexopencv_features2d.hpp:58: error: ‘KAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:59: error: ‘cv::KAZE’ has not been declared
include/mexopencv_features2d.hpp:60: error: ‘cv::KAZE’ has not been declared
include/mexopencv_features2d.hpp:61: error: ‘cv::KAZE’ has not been declared
include/mexopencv_features2d.hpp:65: error: ‘AKAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:66: error: ‘cv::AKAZE’ has not been declared
include/mexopencv_features2d.hpp:67: error: ‘cv::AKAZE’ has not been declared
include/mexopencv_features2d.hpp:68: error: ‘cv::AKAZE’ has not been declared
include/mexopencv_features2d.hpp:72: error: ‘AKAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:73: error: ‘cv::AKAZE’ has not been declared
include/mexopencv_features2d.hpp:74: error: ‘cv::AKAZE’ has not been declared
include/mexopencv_features2d.hpp:75: error: ‘cv::AKAZE’ has not been declared
include/mexopencv_features2d.hpp:79: error: ‘AgastFeatureDetector’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:80: error: ‘cv::AgastFeatureDetector’ has not been declared
include/mexopencv_features2d.hpp:81: error: ‘cv::AgastFeatureDetector’ has not been declared
include/mexopencv_features2d.hpp:82: error: ‘cv::AgastFeatureDetector’ has not been declared
include/mexopencv_features2d.hpp:86: error: ‘AgastFeatureDetector’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:87: error: ‘cv::AgastFeatureDetector’ has not been declared
include/mexopencv_features2d.hpp:88: error: ‘cv::AgastFeatureDetector’ has not been declared
include/mexopencv_features2d.hpp:89: error: ‘cv::AgastFeatureDetector’ has not been declared
include/mexopencv_features2d.hpp:166: error: ‘KAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:166: error: ‘KAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:166: error: template argument 1 is invalid
include/mexopencv_features2d.hpp:168: error: invalid type in declaration before ‘;’ token
include/mexopencv_features2d.hpp:175: error: ‘AKAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:175: error: ‘AKAZE’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:175: error: template argument 1 is invalid
include/mexopencv_features2d.hpp:177: error: invalid type in declaration before ‘;’ token
include/mexopencv_features2d.hpp:184: error: ‘AgastFeatureDetector’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:184: error: ‘AgastFeatureDetector’ is not a member of ‘cv’
include/mexopencv_features2d.hpp:184: error: template argument 1 is invalid
include/mexopencv_features2d.hpp:186: error: invalid type in declaration before ‘;’ token
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::BRISK> createBRISK(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:48: error: no matching function for call to ‘cv::Feature2D::create(std::vector<float, std::allocator<float> >&, std::vector<int, std::allocator<int> >&, float&, float&, std::vector<int, std::allocator<int> >&)’
/usr/local/include/opencv2/features2d/features2d.hpp:269: note: candidates are: static cv::Ptr<cv::Feature2D> cv::Feature2D::create(const std::string&)
src/mexopencv_features2d.cpp:68: error: no matching function for call to ‘cv::Feature2D::create(int&, int&, float&)’
/usr/local/include/opencv2/features2d/features2d.hpp:269: note: candidates are: static cv::Ptr<cv::Feature2D> cv::Feature2D::create(const std::string&)
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::ORB> createORB(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:113: error: no matching function for call to ‘cv::Feature2D::create(int&, float&, int&, int&, int&, int&, int&, int&, int&)’
/usr/local/include/opencv2/features2d/features2d.hpp:269: note: candidates are: static cv::Ptr<cv::Feature2D> cv::Feature2D::create(const std::string&)
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::MSER> createMSER(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:158: error: no matching function for call to ‘cv::FeatureDetector::create(int&, int&, int&, double&, double&, int&, double&, double&, int&)’
/usr/local/include/opencv2/features2d/features2d.hpp:182: note: candidates are: static cv::Ptr<cv::FeatureDetector> cv::FeatureDetector::create(const std::string&)
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::FastFeatureDetector> createFastFeatureDetector(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:183: error: no matching function for call to ‘cv::FeatureDetector::create(int&, bool&, int&)’
/usr/local/include/opencv2/features2d/features2d.hpp:182: note: candidates are: static cv::Ptr<cv::FeatureDetector> cv::FeatureDetector::create(const std::string&)
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::GFTTDetector> createGFTTDetector(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:218: error: no matching function for call to ‘cv::FeatureDetector::create(int&, double&, double&, int&, bool&, double&)’
/usr/local/include/opencv2/features2d/features2d.hpp:182: note: candidates are: static cv::Ptr<cv::FeatureDetector> cv::FeatureDetector::create(const std::string&)
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::SimpleBlobDetector> createSimpleBlobDetector(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:273: error: no matching function for call to ‘cv::FeatureDetector::create(cv::SimpleBlobDetector::Params&)’
/usr/local/include/opencv2/features2d/features2d.hpp:182: note: candidates are: static cv::Ptr<cv::FeatureDetector> cv::FeatureDetector::create(const std::string&)
src/mexopencv_features2d.cpp: At global scope:
src/mexopencv_features2d.cpp:276: error: ‘KAZE’ was not declared in this scope
src/mexopencv_features2d.cpp:276: error: template argument 1 is invalid
src/mexopencv_features2d.cpp: In function ‘int createKAZE(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:287: error: ‘KAZE’ has not been declared
src/mexopencv_features2d.cpp:307: error: ‘KAZE’ has not been declared
src/mexopencv_features2d.cpp: At global scope:
src/mexopencv_features2d.cpp:311: error: ‘AKAZE’ was not declared in this scope
src/mexopencv_features2d.cpp:311: error: template argument 1 is invalid
src/mexopencv_features2d.cpp: In function ‘int createAKAZE(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:317: error: ‘AKAZE’ has not been declared
src/mexopencv_features2d.cpp:323: error: ‘KAZE’ has not been declared
src/mexopencv_features2d.cpp:345: error: ‘AKAZE’ has not been declared
src/mexopencv_features2d.cpp: At global scope:
src/mexopencv_features2d.cpp:349: error: ‘AgastFeatureDetector’ was not declared in this scope
src/mexopencv_features2d.cpp:349: error: template argument 1 is invalid
src/mexopencv_features2d.cpp: In function ‘int createAgastFeatureDetector(__gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:357: error: ‘AgastFeatureDetector’ has not been declared
src/mexopencv_features2d.cpp:371: error: ‘AgastFeatureDetector’ has not been declared
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::FeatureDetector> createFeatureDetector(const std::string&, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:678: error: invalid conversion from ‘int’ to ‘cv::FeatureDetector*’
src/mexopencv_features2d.cpp:678: error:   initializing argument 1 of ‘cv::Ptr<_Tp>::Ptr(_Tp*) [with _Tp = cv::FeatureDetector]’
src/mexopencv_features2d.cpp:680: error: invalid conversion from ‘int’ to ‘cv::FeatureDetector*’
src/mexopencv_features2d.cpp:680: error:   initializing argument 1 of ‘cv::Ptr<_Tp>::Ptr(_Tp*) [with _Tp = cv::FeatureDetector]’
src/mexopencv_features2d.cpp:682: error: invalid conversion from ‘int’ to ‘cv::FeatureDetector*’
src/mexopencv_features2d.cpp:682: error:   initializing argument 1 of ‘cv::Ptr<_Tp>::Ptr(_Tp*) [with _Tp = cv::FeatureDetector]’
src/mexopencv_features2d.cpp: In function ‘cv::Ptr<cv::DescriptorExtractor> createDescriptorExtractor(const std::string&, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >, __gnu_cxx::__normal_iterator<const MxArray*, std::vector<MxArray, std::allocator<MxArray> > >)’:
src/mexopencv_features2d.cpp:713: error: invalid conversion from ‘int’ to ‘cv::DescriptorExtractor*’
src/mexopencv_features2d.cpp:713: error:   initializing argument 1 of ‘cv::Ptr<_Tp>::Ptr(_Tp*) [with _Tp = cv::DescriptorExtractor]’
src/mexopencv_features2d.cpp:715: error: invalid conversion from ‘int’ to ‘cv::DescriptorExtractor*’
src/mexopencv_features2d.cpp:715: error:   initializing argument 1 of ‘cv::Ptr<_Tp>::Ptr(_Tp*) [with _Tp = cv::DescriptorExtractor]’
src/mexopencv_features2d.cpp: At global scope:
src/mexopencv_features2d.cpp:752: error: ‘FLANN_CENTERS_GROUPWISE’ is not a member of ‘cvflann’

    mex: compile of ' "src/mexopencv_features2d.cpp"' failed.

make: *** [lib/mexopencv_features2d.o] Error 1
```

**Comment by kyamagu** [comment]
@hughhugh If you are using old version of OpenCV, use the corresponding release. mexopencv for v2.4 is [here](https://github.com/kyamagu/mexopencv/releases/tag/v2.4.11). Or, v2.3 is available via branch (`git checkout v2.3` before build).

**Comment by hughhugh** [comment]
@kyamagu I use opencv2.4.13, and in the directory of mexopencv, I use "git checkout v2.4" , and `make` command works. However , theres are still some errors when using 'make test'.  What is the solution to the problem? Thank you .
The log as follows:

name@ubuntu:~/mexopencv$ 
name@ubuntu:~/mexopencv$ **make test MATLABDIR=/usr/local/MATLAB/R2013a**
/usr/local/MATLAB/R2013a/bin/matlab -nodisplay -r "addpath(pwd);cd test;try,UnitTest;catch e,disp(e.getReport);end;exit;"

                            < M A T L A B (R) >
                  Copyright 1984-2013 The MathWorks, Inc.
                    R2013a (8.1.0.604) 64-bit (glnxa64)
                             February 15, 2013

 
To get started, type one of these: helpwin, helpdesk, or demo.
For product information, visit www.mathworks.com.
 
== TestANN_MLP ======
-- test_1 ------
Invalid MEX-file '/home/name/mexopencv/+cv/private/ANN_MLP_.mexa64': /usr/local/lib/libopencv_ocl.so.2.4: undefined symbol: _ZTIN2cv16ParallelLoopBodyE

Error in cv.ANN_MLP (line 113)
        this.id = ANN_MLP_();

Error in TestANN_MLP.test_1 (line 10)
            classifier = cv.ANN_MLP([3,1]);

Error in UnitTest.all (line 44)
                    feval([class_name,'.',m{:}]);

Error in UnitTest (line 18)
                UnitTest.all(class_name);

FAIL
== TestAdaptiveThreshold ======
-- test_1 ------
Error using cv.adaptiveThreshold
Invalid MEX-file '/home/name/mexopencv/+cv/adaptiveThreshold.mexa64': /usr/local/lib/libopencv_ocl.so.2.4: undefined symbol: _ZTIN2cv16ParallelLoopBodyE

Error in TestAdaptiveThreshold.test_1 (line 9)
            result = cv.adaptiveThreshold(TestAdaptiveThreshold.img, 255);

Error in UnitTest.all (line 44)
                    feval([class_name,'.',m{:}]);

Error in UnitTest (line 18)
                UnitTest.all(class_name);

FAIL
-- test_2 ------
Error using cv.adaptiveThreshold
Invalid MEX-file '/home/name/mexopencv/+cv/adaptiveThreshold.mexa64': /usr/local/lib/libopencv_ocl.so.2.4: undefined symbol: _ZTIN2cv16ParallelLoopBodyE

Error in TestAdaptiveThreshold.test_2 (line 13)
            result = cv.adaptiveThreshold(TestAdaptiveThreshold.img, 255, 'AdaptiveMethod', 'Gaussian');

Error in UnitTest.all (line 44)
                    feval([class_name,'.',m{:}]);

Error in UnitTest (line 18)
                UnitTest.all(class_name);

FAIL
-- test_3 ------
Error using cv.adaptiveThreshold
Invalid MEX-file '/home/name/mexopencv/+cv/adaptiveThreshold.mexa64': /usr/local/lib/libopencv_ocl.so.2.4: undefined symbol: _ZTIN2cv16ParallelLoopBodyE

Error in TestAdaptiveThreshold.test_3 (line 17)
            result = cv.adaptiveThreshold(TestAdaptiveThreshold.img, 255, 'ThresholdType', 'BinaryInv');

Error in UnitTest.all (line 44)
                    feval([class_name,'.',m{:}]);

Error in UnitTest (line 18)
                UnitTest.all(class_name);

FAIL

**Comment by amroamroamro** [comment]
@hughhugh see #212

**Comment by hughhugh** [comment]
@amroamroamro  Hi，I reinstall matlab without Computer Vision System Toolbox  again, but there are still conflicting problem. Thank you . The log as follows:
== TestIntegral ======
-- test_1 ------
PASS
-- test_2 ------
PASS
-- test_3 ------
PASS
-- test_error_1 ------
PASS
== TestInvert ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
== TestInvertAffineTransform ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
== TestIsContourConvex ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
== TestKNearest ======
-- test_1 ------
PASS
== TestKalmanFilter ======
-- test_1 ------
PASS
== TestKmeans ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
== TestLaplacian ======
-- test_1 ------
PASS
-- test_2 ------
PASS
-- test_error_1 ------
PASS
== TestLatentSvmDetector ======
-- test_1 ------
PASS
== TestLine ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
== TestMSER ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
== TestMatMulDeriv ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
== TestMatchShapes ======
-- test_1 ------
PASS
-- test_error_1 ------
PASS
**and**


root@ubuntu:/home/name/mexopencv# ldd +cv/imread.mexa64
	linux-vdso.so.1 =>  (0x00007fffe189b000)
	libopencv_calib3d.so.2.4 => /usr/local/lib/libopencv_calib3d.so.2.4 (0x00007fafe8c96000)
	libopencv_contrib.so.2.4 => /usr/local/lib/libopencv_contrib.so.2.4 (0x00007fafe89b2000)
	libopencv_core.so.2.4 => /usr/local/lib/libopencv_core.so.2.4 (0x00007fafe8565000)
	libopencv_features2d.so.2.4 => /usr/local/lib/libopencv_features2d.so.2.4 (0x00007fafe82ba000)
	libopencv_flann.so.2.4 => /usr/local/lib/libopencv_flann.so.2.4 (0x00007fafe8040000)
	libopencv_gpu.so.2.4 => /usr/local/lib/libopencv_gpu.so.2.4 (0x00007fafe7df1000)
	libopencv_highgui.so.2.4 => /usr/local/lib/libopencv_highgui.so.2.4 (0x00007fafe7b9c000)
	libopencv_imgproc.so.2.4 => /usr/local/lib/libopencv_imgproc.so.2.4 (0x00007fafe76e0000)
	libopencv_legacy.so.2.4 => /usr/local/lib/libopencv_legacy.so.2.4 (0x00007fafe73bc000)
	libopencv_ml.so.2.4 => /usr/local/lib/libopencv_ml.so.2.4 (0x00007fafe713b000)
	libopencv_nonfree.so.2.4 => /usr/local/lib/libopencv_nonfree.so.2.4 (0x00007fafe6f02000)
	libopencv_objdetect.so.2.4 => /usr/local/lib/libopencv_objdetect.so.2.4 (0x00007fafe6c7c000)
	libopencv_ocl.so.2.4 => /usr/local/lib/libopencv_ocl.so.2.4 (0x00007fafe6896000)
	libopencv_photo.so.2.4 => /usr/local/lib/libopencv_photo.so.2.4 (0x00007fafe6677000)
	libopencv_stitching.so.2.4 => /usr/local/lib/libopencv_stitching.so.2.4 (0x00007fafe63f8000)
	libopencv_superres.so.2.4 => /usr/local/lib/libopencv_superres.so.2.4 (0x00007fafe61bc000)
	libopencv_video.so.2.4 => /usr/local/lib/libopencv_video.so.2.4 (0x00007fafe5f6a000)
	libopencv_videostab.so.2.4 => /usr/local/lib/libopencv_videostab.so.2.4 (0x00007fafe5d31000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fafe5b29000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fafe590c000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fafe5707000)
	libmx.so => not found
	libmex.so => not found
	libmat.so => not found
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fafe540a000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fafe510a000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fafe4ef3000)

**Comment by amroamroamro** [comment]
@hughhugh 
The unit tests you're showing all passed, right?

Now in order to find conflicting libraries, you must run `ldd` both in the MATLAB command prompt and in the system shell, and look for differences in loaded libraries. 

You didn't really need to uninstall CVST toolbox (that would be a last resort). You could have either temporarily moved the `libopencv_*.so.2.4` libs inside the MATLAB bin directory, replace symlinks, or used the `LD_PRELOAD` preloading trick explained in the previous links I posted...

Alternatively you could move to latest OpenCV 3.1.0, avoiding conflict with MATLAB's included OpenCV 2.4.x. That way you can keep using both mexopencv and CVST at the same time.

**Comment by kyamagu** [comment]
> Did you clean the previous build artifacts? i.e., make clean

if that doesn't solve the issue, checkout the old tag.
